components/husky_remove_miniitx.mdx

```diff
@@ -0,0 +1,41 @@
+1. If you have a mini-ITX computer installed it will need to be removed. Locate the power and
+   communication cable connected to the computer and remove them. If you have any other sensors attached,
+   remove those too.  Keep them connected to the Husky plaform.
+
+    <center>
+      <figure>
+        <img
+          src="/img/computer_images/husky_pc_removal/1.jpg"
+          width="800"
+        />
+        <figcaption>Disconnecting cables</figcaption>
+      </figure>
+    </center>
+
+2. Remove the four (4) hex screws from the computer case using a 2.5mm wrench.
```

Comment:
- Change to ' Remove the 4 hex screws...'
- Change to '2.5 mm'

Reply:
Done